schema/src/main/jade-tables/hles_dog.table.json

```diff
@@ -17,6 +17,26 @@
         }
       ]
     },
+    {
+      "name": "st_vip_or_staff",
+      "datatype": "string"
```

Comment:
I think I might have been getting ahead of the updates to the mapping doc... I'm not sure if this column should actually be an integer.

Reply:
Yes, you got ahead of me! This one should be an int, the rest should be ok as-is.